source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,209 @@
+ideas and innovations upstream through the Linux kernel and 
+using it to take advantage of the latest and greatest features. 
+This includes security-centered innovations adopted in the Linux upstream. 
+
+ 
+
+Automated Effective Updating
+----------------------------
+
+The |CL| upstream is updated at lightning pace, releasing full
+new versions of |CL| with incremental updates multiple times a day! 
+
+This `rolling release model`_ allows |CL| to consume the latest security
+fixes of software packages as soon as they become available. 
+There is no waiting for major or minor releases on |CL|. 
+
+But an update is not effective if it is just simply downloaded onto a system. 
+It needs to be obtained *AND* ensured that the new patched copy is being
+used; not an older copy loaded into memory. |CL| will let you know when a 
+service needs to be rebooted or do it for your automatically after 
+a software update, if desired.
+
+
+In |CL| updates are delivered automatically, efficiently, 
+and effectively.For more information see `documentation about Software Updates`_ in |CL|.
+
+ 
+
+
+
+Automated CVE Scanning and Remediation
+--------------------------------------
+
+The sheer number of software packages and security vulnerabilities is growing 
+at an exorbitant rate. Repositories of CVEs and their fixes, if known, 
+are published by :abbr:`NIST` in a National Vulnerability Database 
+\ |NVD|\  and at \ |MITRE|\  .
+
+
+|CL| takes a proactive and measured approach to addressing known 
+and fixable Common Vulnerabilities and Exposures (CVEs) disclosures.
+|CL| packages are scanned against CVEs at least daily, and 
+software patched in accordance to the fix. This scanning is automated 
+with an open source tool - `cve-check-tool`_ - and any outstanding
+vulnerabilities monitored appropriately.
+
+
+These combined practices minimize the amount of 
+time |CL| systems are exposed to unnecessary security risk.
+
+ 
+
+
+
+Security in Software
+====================
+
+
+Minimized attack surface
+-------------------------
+
+|CL| removes legacy, unneeded, or redundant standards and
+components as much as possible to enable the use of best known security 
+standards. Below are some examples: 
+
+* `RC4`, `SSLv3`, `3DES`, and `SHA-1` ciphers which have had known 
+  vulnerabilities, have been explicitly disabled within many |CL| packages to 
+  avoid their accidental usage. 
+
+* Services and subsystems which expose sensitive system information 
+  have been removed such as the `finger` and `tcpwrappers`.
+
+* `SFTP` has been disabled by default due to security 
+  considerations. See the `openssh-server reference page`_ for more details. 
+
+
+Verified trust
+--------------------------
+
+|CL| encourages the use of secure practices such as encryption
+and digital signature verification throughout the system and discourages blind
+trust. Below are some examples: 
+
+* All update operations from swupd are transparently encrypted and checked 
```

Comment:
Remove "transparently" ?

Reply:
Actually - let's rework this paragraph together.

Reply:
@puneetse. Did you guys resolve this?

Reply:
Hi @nesiusra, unless you have more comments here about the wording, I'd like to publish this within the next 24 hrs.  In case you see this later than that, you can always follow up with me, and we can do a new PR. Thanks.